core/com.b2international.snowowl.core/src/com/b2international/snowowl/core/codesystem/CodeSystem.java

```diff
@@ -35,6 +35,10 @@
 	
 	public static final String RESOURCE_TYPE = "codesystems";
 	
+	//CodeSystem settings
+	public static final String DEFAULT_REFSET_MODULE_ID = "defaultRefsetModuleId";
```

Comment:
These are already defined in the `SnomedTerminologyComponentConstants` class, please remove them.